cmd/translatesfx/README.md

```diff
@@ -116,3 +130,343 @@ service:
       exporters:
       - signalfx
 ```
+
+## Translatable Features
```

Comment:
"Translatable Features" to "Translatable features"

Reply:
Done.